### PR TITLE
fred good dog

### PR DESCRIPTION
**Pull request by ZhaoYuLL**

### CODE REVIEW

**Review by pak-plau — DISMISSED** [review]
remove the finals from the variables, you don't need them.

**Review by rmui1 — CHANGES_REQUESTED** [review]
Also, can you delete the Dog.class file?

**Comment by ZhaoYuLL** [comment]
removed finals, deleted Dog.class

File: ZhaoYuLin/Dog.java [hunk 1]
```diff
@@ -6,19 +6,50 @@
...
+        System.out.println("Fred's age: " + Fred.getAge());
+        Fred.setRabies(true);
+          Dog bar = new Dog("bar", "dachshund",1);
+        System.out.println("Bar's age in dog years: " + bar.getAgeinDogyears());
+        bar.setRabies(true);
+        System.out.println("Does bar have rabies: " + bar.hasRabies());
+
+
+    }
+    public Dog(){
```
Comment (rmui1) [inline ZhaoYuLin/Dog.java]: In general, your constructors should go together near the top. Put this constructor right above the other one.

File: ZhaoYuLin/Dog.java [hunk 2]
```diff
@@ -6,19 +6,50 @@
...
+
     public void bark() {
         System.out.println("bork");
     }
-    
+
     public static void main(String[] args) {
         Dog foo = new Dog("foo", "golden retriever", 5);
         foo.bark();
+        Dog Fred = new Dog();
```
Comment (rmui1) [inline ZhaoYuLin/Dog.java]: Object names are usually in camel case, meaning the first word is all lower case and then the following words have their first letter capitalized. 
Ex: dog, bigDog, veryBigDog, etc.

Fix the object name here.

File: ZhaoYuLin/Dog.java [hunk 3]
```diff
@@ -6,19 +6,50 @@
...
+        Fred.bark();
+        System.out.println("Fred's age: " + Fred.getAge());
+        Fred.setRabies(true);
+          Dog bar = new Dog("bar", "dachshund",1);
```
Comment (rmui1) [inline ZhaoYuLin/Dog.java]: Get rid of the extra space here, so line 27 is in line with everything else.

File: ZhaoYuLin/Dog.java [hunk 4]
```diff
@@ -6,19 +6,50 @@
...
         rabies = false;
     }
-   
+
     public void bark() {
         System.out.println("bork");
     }
-    
+
     public static void main(String[] args) {
```
Comment (rmui1) [inline ZhaoYuLin/Dog.java]: We usually try to keep all the methods (not including the main method) together. Either move the main method below all the other methods (preferred) or move it above them, but after the constructors.

File: ZhaoYuLin/Dog.java [hunk 5]
```diff
@@ -6,19 +6,50 @@
...
+        System.out.println("Does bar have rabies: " + bar.hasRabies());
+
+
+    }
+    public Dog(){
+        name = "Fred";
+        breed = "Husky";
+        age = 3;
+        rabies = true;
+
```
Comment (rmui1) [inline ZhaoYuLin/Dog.java]: Get rid of the two new lines here.

File: ZhaoYuLin/Dog.java [hunk 6]
```diff
@@ -6,19 +6,50 @@
...
+        age = 3;
+        rabies = true;
+        
+    
+    }
+    public int getAge(){
+        return age;
+    }
+    public int getAgeinDogyears(){
+        return (age + 7);
```
Comment (rmui1) [inline ZhaoYuLin/Dog.java]: It should be multiplied by 7.

**Review by rmui1 — CHANGES_REQUESTED** [review]
Oops, sorry here are just a few more changes. Your code is really great, but here are just a few suggestions so that your code follows convention. 

Thank you so much!

**Review by rmui1 — APPROVED** [review]

**Pull request merged**